python/rapidsmpf/rapidsmpf/memory/buffer_resource.pyx

```diff
@@ -188,6 +192,7 @@ cdef class BufferResource:
         with nogil:
             self._handle = make_shared[cpp_BufferResource](
                 device_mr.get_mr(),
+                nullptr,  # TODO: Write Python bindings for PinnedMemoryResource
```

Comment:
Answering a question: I guess at the moment this won't break cudf-polars with rapidsmpf since we're using the Python bindings and those haven't changed (yet).